test/functional/mempool_accept.py

```diff
@@ -90,9 +90,17 @@ def run_test(self):
         txid_in_block = self.wallet.sendrawtransaction(from_node=node, tx_hex=raw_tx_in_block)
         self.generate(node, 1)
         self.mempool_size = 0
+        # Also check feerate. 1BTC/kvB fails
+        assert_raises_rpc_error(-8, "Fee rates larger than or equal to 1BTC/kvB are not accepted", lambda: self.check_mempool_result(
```

Comment:
nit (fa2a4fdef779b01e847def5985deafedc6dd3da8): We could test negative values as well.

Reply:
Created a PR to cover this https://github.com/bitcoin/bitcoin/pull/29459